src/scmdata/run.py

```diff
@@ -885,7 +891,7 @@ def values(self) -> np.ndarray:
             The array in the same shape as :py:obj:`ScmRun.shape`, that is
             ``(num_timeseries, num_timesteps)``.
         """
-        return np.asarray([ts._data.values for ts in self._ts])
+        return self._df.values.T
```

Comment:
should we just store the data internally so we don't need the `.T`? Holding it transposed internally seems a confusing route to go down.

Reply:
I'll take a quick look, but can happen in another MR.

Another thing to think about is if we need the Multiindex vs a DataFrame of dtype "category". There are quite a few times where we switch back and forward. Then the index of meta can always be the same as the index of the data. We lose that info by converting to/from a MultiIndex and rely on order

It would be interesting to know what actually happens when `pd.MultiIndex.from_frame(df._meta.to_frame().reset_index(drop=True))` is called. Anecdotally it seems pretty fast.

Reply:
ye nice, both good refactorings for future potentially